README.md

```diff
@@ -80,11 +80,14 @@ Now, the MJPEG stream could be opened by pointing a browser to [http://localhost
 Usage of bin/amd64/generic-device-plugin:
       --device stringArray        The devices to expose. This flag can be repeated to specify multiple device types.
                                   Multiple paths can be given for each type. Paths can be globs.
-                                  Should be provided in the form:
```

Comment:
This line should not have been deleted. Can you try regenerating this file using `make README.md` CI should complain about this

Reply:
I don't presently have a working local docker setup; is this something you can easily do or should I keep yelling at the docker client?

Reply:
https://github.com/squat/generic-device-plugin/pull/3
Hope this helps :)) once this merges, rebase and you should be able to do all of the make stuff, like `CONTAINERIZE_BUILD=false make README.md`

Reply:
Awesome, thanks! Resolved in 8b12e5c